hles/extraction/src/main/scala/org/broadinstitute/monster/dap/EnvironmentExtractionPipeline.scala

```diff
@@ -24,7 +24,23 @@ object EnvironmentExtractionPipeline extends ScioApp[Args] {
   val subdir = "environment"
   //todo: need to query for all arms and work through arms serially
   val arm =
-    List("baseline_arm_1", "dec2019_arm_1", "jan2020_arm_1")
+    List(
+      "baseline_arm_1",
+      "dec2019_arm_1",
+      "dec2019_secondary_arm_1",
+      "jan2020_arm_1",
+      "jan2020_secondary_arm_1",
+      "feb2020_arm_1",
+      "feb2020_secondary_arm_1",
+      "mar2020_arm_1",
+      "mar2020_secondary_arm_1",
+      "apr2020_arm_1",
+      "apr2020_secondary_arm_1",
+      "may2020_arm_1",
+      "may2020_secondary_arm_1",
+      "june2020_arm_1",
+      "june2020_secondary_arm_1"
```

Comment:
❓ Does this only go to June 2020 for performance reasons?

Reply:
yes - adding any more arms makes this fail when running a full refresj

Reply:
do we have any mechanisms to throttle this to let us add all of the arms we need without beating redcap into submission? this seems like a high priority feature to add if it's stopping us from doing a proper full refresh.

Reply:
No, there is no such mechanism now; @quazi-broad and I have been talking about this, we'll tackle it after this gets merged (@quazi-broad can you make sure there's a ticket for this and link it here?)

Reply:
https://broadinstitute.atlassian.net/browse/DSPDC-1415